src/utils/mtev_zipkin_curl.h

```diff
@@ -37,11 +37,42 @@
 
 static inline struct curl_slist *
 mtev_zipkin_inst_curl_headers(struct curl_slist *inheaders) {
+  static const char *curl_name = "curl request";
```

Comment:
Technically we don't have the URL yet when that call is made.  But the URL is already passed as a binary annotation.  I could provide an alternative form of that function for use with the naming of curl requests arbitrarily.